src/Meshes/geometry.ts

```diff
@@ -704,7 +716,7 @@ export class Geometry implements IGetSetVerticesData {
 
             if (kind === VertexBuffer.PositionKind) {
                 if (!this._extend) {
-                    this._updateExtend();
+                    this._updateExtend(null, stride);

```

Comment:
Same comment than above, the `stride` value can be computed from `this._vertexBuffers[kind].byteStride`.